### ISSUE TITLE
Pair Wii Remote in Priiloader Menu

### ISSUE THREAD

**Issue by DraconicNEO** [DacoTaco/priiloader#277]
I think the ability to pair Wii remotes should be added to Priiloader so people can Pair a Wii remote without having to boot to the system menu. This would be useful in sticky situations where you are only able to boot to Priiloader but don't have a way to pair the remotes or any other way of controlling Priiloader without a Wii remote.

**Comment by TheTechRobo** [comment]
Yes, good idea. I think was suggested a while ago.

**Comment by DraconicNEO** [comment]
> 
> 
> Yes, good idea. I think was suggested a while ago.

Really? I looked around but couldn't find an Issue for it.

**Comment by TheTechRobo** [comment]
Yeah, maybe no Issue. But probably on gbatemp. 

Was just saying, very good idea, lots of poep,e would benefit. :)

**Comment by DacoTaco** [comment]
i'd love to add that possibility, but this is sadly a libogc/wiiuse issue.
afaik, no homebrew can sync because of this

**Comment by TheTechRobo** [comment]
Out of curiosity, what's the issue?

**Comment by DacoTaco** [comment]
i have no idea to be honest,i never looked into it very deeply as wiiuse code a thing on its own xD
i think its not responding or doing something right with the sync messages/events.

i got a very old homebrew proof of concept on my pc that can sync wiimotes, but i never did find out where the issue lies.
needless to say, if homebrew could sync wii's it would be huge hehe

**Comment by TheTechRobo** [comment]
@DacoTaco Yeah, would definitely be awesome if it could sync wiimotes. Hopefully the issue gets fixed sometime

**Comment by Leseratte10** [comment]
> i got a very old homebrew proof of concept on my pc that can sync wiimotes, but i never did find out where the issue lies.

@DacoTaco Do you still have that homebrew somewhere, maybe even with source code? Can you post that somewhere so others can take a look at it? I'd be interested in adding wiimote sync support to homebrew, too.

**Comment by TheTechRobo** [comment]
I second Leseratte's request. Having that homebrew might really help

**Comment by DacoTaco** [comment]
did some digging. the app has everything build in except for the bluetooth stack, which was build back in 2008ish against libogc.
however, by the time libogc go wiiuse , [it reworked its bluetooth stack to support wiiuse better](https://github.com/devkitPro/libogc/commit/0fa1eb17c34676161ae2c6327e5a2e08f8eb668f), but it seems the integration wasn't completed. there are missing functions & wiiuse integrations that still needed to be done. i got the app to build, but the bluetooth/wiiuse integration of said app is still completely broken.

**Comment by MasterLuma** [comment]
I agree that the ability to Sync Wiimotes in Priiloader would be a nice and possibly life-saving feature for a lot of people. However, GameCube controllers work perfectly in Priiloader, and if someone is in a dire situation where they need to control Priiloader's options, they could always just use that instead. But I understand that not everyone has or wants to bother with finding a GC controller.

**Comment by TheTechRobo** [comment]
Also the latest Priiloader releases support navigating with the front panel buttons like BootMii does

**Comment by DacoTaco** [comment]
and keyboards also work :p 

i still need to look into this though.. but its all very new to me and i lack the time to do it

**Comment by DacoTaco** [comment]
i can close this, as any homebrew to be build with latest libogc can now sync wiimotes!
